roles/common/tasks/main.yml

```
@@ -11,6 +11,12 @@
   include_vars: "nexus.yml"
   failed_when: false
 
+- name: Ensure apt indices are up-to-date
+  apt:
+    update_cache: yes
+    cache_valid_time: 3600 # 1 hour
+  when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu'
```

There is a dediacted fact for that:

```suggestion
  when: ansible_pkg_mgr == 'apt'
```